### ISSUE TITLE
[subviews] Visual glitch in PST crystal on AR1202

### ISSUE THREAD

**Issue by MarcelHB** [gemrb/gemrb#775]
#### Bug description
Look at the screenshot below (SDL2 in use).

#### Steps to reproduce
1. In PST, go to AR1202.

#### Screenshots
master on top, subviews below.

![pst_crystal](https://user-images.githubusercontent.com/238558/83699496-79b1d900-a604-11ea-8352-d305a34c2b5e.jpg)

**Comment by bradallred** [comment]
definitely worked at some point

**Comment by bradallred** [comment]
I'm not seeing this. was it fixed by #771?

**Comment by MarcelHB** [comment]
I still get it. I will switch the SDL2 backend in a minute.

**Comment by bradallred** [comment]
I checked both and I don't see it in either...

**Comment by MarcelHB** [comment]
The only thing I'm aware of right now. Anyway, I see it on both D3D and OpenGL here. I will try to mta there from a new game since we worked on this a year ago.

**Comment by MarcelHB** [comment]
No luck, I know it worked once upon a time. When I'm done reporting stuff, I will do bisecting.

**Comment by bradallred** [comment]
I suspect the breakage would be recent. I know this was working at some point in working on #441

**Comment by bradallred** [comment]
[deja vú](https://github.com/gemrb/gemrb/issues/441#issuecomment-563411812)

**Comment by MarcelHB** [comment]
Oh right, but let's keep the issue unless it's a duplicate of some other problem.

**Comment by bradallred** [comment]
I have no intention of closing this issue and the problem _was_ fixed (and still is for me?)

Can you bisect from the time of that comment to now and see if you can find when it was fixed and when it was re-broken?

**Comment by MarcelHB** [comment]
Git bisect says 002befaf80f20b60151856ae63648ddfccf1e334 has introduced that behavior.

**Comment by bradallred** [comment]
Your comment was from a week prior which suggests that it was broken before that commit, so if that commit broke it then it would be useful to know what commit apparently fixed it previously.

**Comment by bradallred** [comment]
also, since it apparently "works for me" I wonder if it is dependent on settings like the "Always dither" setting or something.

**Comment by bradallred** [comment]
ok, it appears you are using _only_ SDL 2 and not SDL2+OpenGL. This will never work without OpenGL as noted #441 "base" SDL 2 is technologically limited to using only one channel.

We can adjust it so `BLIT_STENCIL_GREEN` (the area animation channel) results in no blending for the "base" SDL 2 driver which should look much better in this case (probably would look 100% correct in fact), but that would make `AreaAnimations` that are supposed to be covered by a wall uncovered. So I'm not sure which is better.

TL;DR: If you want 100% correct stenciling everywhere you must use SDL2+OpenGL or SDL1.

**Comment by MarcelHB** [comment]
Right, I start to remember that there was something left to think of.

**Comment by bradallred** [comment]
assuming then this does work for you with`OPENGL_BACKEND=1` I shall close the issue
